src/components/Background.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import '../index.css';
+
+class Background extends Component {
+  render() {
+    return (
+
+
+
+        <div className = "bg2">
+          <img className = 'img2' src ={require('../images/PoroLogo.png')} />
```

Comment:
change to "logo" and turn into `<img className="pororift-logo" />`. Remember spaces usually separate the attributes in the elements. looks a little confusing when you see it in between the =